### DISCUSSION TITLE
zotero-reference下载

### DISCUSSION THREAD

**Discussion by taskkk** [MuiseDestiny/zotero-reference#193] [General]
请问zotero-reference该如何下载呢？谢谢！！

**Comment by MuiseDestiny** [comment]
![image](https://github.com/MuiseDestiny/zotero-reference/assets/51939531/23b2e700-092d-4ac8-9dcf-e721de89287c)
![image](https://github.com/MuiseDestiny/zotero-reference/assets/51939531/11e75dd8-9952-4993-8a1f-aa9d20a8ff41)

**Comment by taskkk** [comment]
收到，十分感谢！！



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Polygon ***@***.***&gt;
发送时间: 2023年9月7日 10:37
收件人: MuiseDestiny/zotero-reference ***@***.***&gt;
抄送: sijin720 ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [MuiseDestiny/zotero-reference] zotero-reference下载 (Discussion #193)
